docs/docs/stack_config.md

```diff
@@ -5,172 +5,227 @@ title: Stack Config
+    - "value2"
+  <parameter4_name>:
+    - !<resolver_name> <resolver_value>
+    - !<resolver_name> <resolver_value>
+  <parameter5_name>:
+    - !<resolver_name> <resolver_value>
+    - "value1"
 ```
 
 Example:
 
 ```yaml
 parameters:
-    database_username: "mydbuser"
-    database_password: !environment_variable DATABASE_PASSWORD
-    subnet_ids:
-        - "subnet-12345678"
-        - "subnet-87654321"
-    security_group_ids:
-        - "sg-12345678"
-        - !stack_output security-groups::BaseSecurityGroupId
-        - !file_contents /file/with/security_group_id.txt
+  database_username: "mydbuser"
+  database_password: !environment_variable DATABASE_PASSWORD
+  subnet_ids:
+    - "subnet-12345678"
+    - "subnet-87654321"
+  security_group_ids:
+    - "sg-12345678"
+    - !stack_output security-groups::BaseSecurityGroupId
+    - !file_contents /file/with/security_group_id.txt
 ```
 
 ### protect
 
 Stack protection against execution of the following commands:
 
-- `launch-stack`
-- `create-stack`
-- `update-stack`
-- `delete-stack`
-- `execute-change-set`
+- `launch`
+- `create`
+- `update`
+- `delete`
+- `execute`
 
-If a user tries to run one of these commands on a protected stack, Sceptre will throw an error.
+If a user tries to run one of these commands on a protected Stack, Sceptre will
+throw an error.
 
-### sceptre\_user\_data
+### sceptre_user_data
 
-Represents data to be passed to the `sceptre_handler(sceptre_user_data)` function in Python templates or accessible under `sceptre_user_data` variable key within Jinja2 templates.
+Represents data to be passed to the `sceptre_handler(sceptre_user_data)`
+function in Python templates or accessible under `sceptre_user_data` variable
+key within Jinja2 templates.
 
 ### stack_name
 
 A custom name name to use instead of the Sceptre default.
 
 <div class="alert alert-warning">
-Outputs from stacks with custom names can't be resolved using the standard <a href="{{ site.baseurl }}/docs/resolvers.html#stack_output">stack output resolver</a>. Outputs should be resolved using the <a href="{{ site.baseurl }}/docs/resolvers.html#stack_output_external">stack output external resolver</a>. An explicit dependency should be added, using the <a href="#dependencies">dependencies</a> parameter, to make sure the stacks are launched in the correct order.
+Outputs from Stacks with custom names can't be resolved using the standard <a
+href="{{ site.baseurl }}/docs/resolvers.html#stack_output">stack output
+resolver</a>. Outputs should be resolved using the <a href="{{ site.baseurl
+}}/docs/resolvers.html#stack_output_external">stack output external
+resolver</a>. An explicit dependency should be added, using the <a
+href="#dependencies">dependencies</a> parameter, to make sure the Stacks are
+launched in the correct order.
 </div>
 
 e.g:
 
 ```yaml
 parameters:
-    VpcID: !stack_output_external <custom-named-vpc-stack>::VpcID
+  VpcID: !stack_output_external <custom-named-vpc-stack>.yaml::VpcID
```

Comment:
perhaps another example mentioning the optional 2nd arg to `stack_output_external` which can be the profile

If the template you're referring to is in a different AWS account or region, then this can be accessed by supplying an optional and valid AWS profile:
```yaml
VpcID: !stack_output_external <custom-named-vpc-stack>.yaml::VpcID my-aws-prod-profile
```

Reply:
👍